doc/getting_started/installation_win.rst

```diff
@@ -20,6 +20,144 @@ Windows system with the latest updates installed.
 Installing Requirements and Dependencies
 ****************************************
 
+There are 3 different ways of developing for Zephyr on Microsoft Windows.
+The first one is fully Windows native, whereas the 2 additional ones require
+emulation layers that slow down build times and are not as optimal. All of
+them are presented here for completeness, but unless you have a particular
+requirement for a UNIX tool that is not available on Windows, we strongly
+recommend you use the Windows Command Prompt for performance and minimal
+dependency set.
+
+Using Windows Command Prompt (Recommended, fastest)
+===================================================
+
+The easiest way to install the dependencies natively on Microsoft Windows is
+to use the :program:`Chocolatey` package manager (`Chocolatey website`_).
+If you prefer to install those manually then simply download the required
+packages from their respective websites.
+
+   .. note::
+        There are multiple ``set`` statements in this tutorial. You can avoid
+        typing them every time by placing them inside a ``.cmd`` file and
+        running that every time you open a Command Prompt.
+
+#. If you're behind a corporate firewall, you'll likely need to specify a
+   proxy to get access to internet resources::
+
+      > set HTTP_PROXY=http://user:password@proxy.mycompany.com:1234
+
+#. Install :program:`Chocolatey` by following the instructions on the
+   `Chocolatey install`_ website.
+
+#. Open a Command Prompt (`cmd.exe`) as an **Administrator**.
+
+#. Optionally disable global confirmation to avoid having to add `-y` to all
+   commands:
+
+   .. code-block:: console
+
+      > choco feature enable -n allowGlobalConfirmation
+
+#. Install CMake and DTC:
+
+   .. code-block:: console
+
+      > choco install cmake --installargs 'ADD_CMAKE_TO_PATH=System'
+      > choco install dtc-msys2 --version 1.4.4
+
+#. Install the rest of the tools:
+
+   .. code-block:: console
+
+      > choco install git python ninja gperf
+
+#. Close the Command Prompt window.
+
+#. Open a Command Prompt (`cmd.exe`) as a **regular user**.
+
+#. Clone a copy of the Zephyr source into your home directory using Git.
+
+   .. code-block:: console
+
+      > cd %userprofile%
+      > git clone https://github.com/zephyrproject-rtos/zephyr.git
+
+#. Install the required Python modules::
+
+      > cd %userprofile%\zephyr
+      > pip install --user -r scripts/requirements.txt
+
+#. The build system should now be ready to work with any toolchain installed in
+   your system. In the next step you'll find instructions for installing
+   toolchains for building both x86 and ARM applications.
+
```

Comment:
At this point I tried to generate the documentation ( > make htmldocs ) and it failed because ``make`` wasn't installed (looks like that gets installed with the next step for the ISSM Toolchain), but i wonder if other build system tool chains will install ``make`` as well.  Even with ``make`` installed, the python script folder (where sphinx-build.exe is installed) isn't added to the ``%path%`` so the doc build fails.

Reply:
Building the documentation requires `make` which really is not part of the Windows command set. We used to have this in MSYS2, but now the goal is to use only Windows native tools (unless the toolchain itself comes with them, but that's not part of the Zephyr core toolset). I suggest we switch to CMake for building the docs or specify that they can only be built on Linux, macOS and MSYS2. @nashif, @SebastianBoe, thoughts?

Reply:
Maybe one of you cmake wizards can use this info (https://eb2.co/blog/2012/03/sphinx-and-cmake-beautiful-documentation-for-c---projects/) to switch the doc build over to Cmake?

Reply:
how to build documentation on windows is another patch for another PR, lets not block this PR because we can't build documentation.

Reply:
@dbkinder I’ll give it a go in a separate PR

Reply:
Thanks.